src/mahoji/lib/abstracted_commands/naxxusCommand.ts

```diff
@@ -0,0 +1,238 @@
+import { MessageEmbed } from 'discord.js';
+import { calcPercentOfNum, calcWhatPercent, increaseNumByPercent, reduceNumByPercent } from 'e';
+import { KlasaUser } from 'klasa';
+import { Bank } from 'oldschooljs';
+import { Item } from 'oldschooljs/dist/meta/types';
+
+import { checkUserCanUseDegradeableItem, degradeableItems, degradeItem } from '../../../lib/degradeableItems';
+import { GearStats } from '../../../lib/gear';
+import { Naxxus } from '../../../lib/minions/data/killableMonsters/custom/bosses/Naxxus';
+import { trackLoot } from '../../../lib/settings/prisma';
+import { ClientSettings } from '../../../lib/settings/types/ClientSettings';
+import { UserSettings } from '../../../lib/settings/types/UserSettings';
+import { Gear } from '../../../lib/structures/Gear';
+import { NaxxusActivityTaskOptions } from '../../../lib/types/minions';
+import { formatDuration, isWeekend, updateBankSetting } from '../../../lib/util';
+import addSubTaskToActivityTask from '../../../lib/util/addSubTaskToActivityTask';
+import getOSItem from '../../../lib/util/getOSItem';
+
+const bisMageGear = new Gear({
+	head: 'Gorajan occult helmet', // 20
+	body: 'Gorajan occult top', // 45
+	legs: 'Gorajan occult legs', // 40
+	hands: 'Gorajan occult gloves', // 22
+	feet: 'Gorajan occult boots', // 22
+	cape: 'Vasa cloak', // 38
+	ring: 'Spellbound ring(i)', // 25
+	weapon: 'Void staff', // 250
+	shield: 'Abyssal tome', // 45
+	neck: 'Arcane blast necklace' // 25
+});
+
+const bisMeleeGear = new Gear({
+	head: 'Gorajan warrior helmet',
+	body: 'Gorajan warrior top',
+	legs: 'Gorajan warrior legs',
+	hands: 'Gorajan warrior gloves',
+	feet: 'Gorajan warrior boots',
+	neck: "Brawler's hook necklace",
+	cape: 'TzKal cape',
+	weapon: 'Drygore rapier',
+	shield: 'Offhand drygore rapier',
+	ring: 'Ignis ring(i)'
+});
+
+const itemBoosts: {
+	item: Item;
+	boost: number;
+	setup: 'mage' | 'melee';
+}[] = [
+	{
+		item: getOSItem('Void staff'),
+		boost: 10,
+		setup: 'mage'
+	},
+	{
+		item: getOSItem('Abyssal tome'),
+		boost: 7.5,
+		setup: 'mage'
+	},
+	{
+		item: getOSItem('Spellbound ring(i)'),
+		boost: 2.5,
+		setup: 'mage'
+	},
+	{
+		item: getOSItem('Spellbound ring'),
+		boost: 2,
+		setup: 'mage'
+	}
+];
+
+const NAXXUS_HP = 3900;
```

Comment:
this value is stored in 2 places. better to try consolidate it into 1 for consistency, so we dont accidentally change only 1